app/src/main/res/drawable/ic_logo_foreground.xml

```diff
@@ -0,0 +1,40 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="108.3"
+    android:viewportHeight="110.15">
+    <path
+        android:fillColor="#1976d2"
+        android:pathData="M0.15,0.15h108v108h-108z"
+        android:strokeWidth="0.3"
+        android:strokeColor="#000" />
+    <path android:pathData="M38.15,75.78C47.39,86.55 70.56,110.15 70.56,110.15l34.35,-28.67L69.64,45.71 69.15,63.04l-5.27,0.47 0,5.15L47.44,68.66l0.55,7.1Z">
+        <aapt:attr name="android:fillColor">
+            <gradient
+                android:endX="123.15"
+                android:endY="101.14"
+                android:startX="32.15"
+                android:startY="61.33"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/8067b306ac65fa80339c37fd04f62558cec7eeed/app/src/main/res/drawable/ic_logo_foreground.xml#L18"></span>Attribute <code>startY</code> is only used in API level 24 and higher (current min is 21)</td>
    </tr>
  </tbody>
</table>